src/main/scala/io/shiftleft/js2cpg/preprocessing/TranspilationRunner.scala

```diff
@@ -23,6 +23,8 @@ class TranspilationRunner(projectPath: Path, tmpTranspileDir: Path, config: Conf
 
   private val transpilers: Seq[Transpiler] = createTranspilers()
 
+  private val DEPS_TO_KEEP: List[String] = List("@vue", "vue", "nuxt", "@babel")
```

Comment:
```suggestion
  private val DEPS_TO_KEEP: Seq[String] = Seq("@vue", "vue", "nuxt", "@babel")
```

not that it really matters

Reply:
Ordering or performance (4 elements) is really not important here, yes.